File: Day43/DSA/Find_Words_Containing_Character.java
Question:-  https://leetcode.com/problems/find-words-containing-character/description/

Code:-

class Solution {
    public List<Integer> findWordsContaining(String[] words, char x) {

        int n = words.length;

        ArrayList<Integer> list = new ArrayList<>();

        for(int i = 0;i<n;i++)
        {
            if(words[i].indexOf(x) != -1)
            {
                list.add(i);
            }
        }

        return list;
    }
}

File: Day43/DSA/Find_words_containing_character.cpp
Question:-  https://leetcode.com/problems/find-words-containing-character/description/


Code:-  

class Solution {
public:
    vector<int> findWordsContaining(vector<string>& words, char x) {
        vector<int> v;

        int n = words.size();

        for (int i = 0; i < n; i++) {
            for (int j = 0; j < words[i].size(); j++) {
                if (words[i][j] == x) {
                    v.push_back(i);
                    break;
                }
            }
        }

        return v;
    }
};